pkgs/os-specific/darwin/itsycal/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchzip
+, stdenvNoCC
+, unzip
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "itsycal";
+  version = "0.15.3";
+
+  src = fetchzip {
+    url = "https://itsycal.s3.amazonaws.com/Itsycal-${finalAttrs.version}.zip";
+    hash = "sha256-jpTlJY7yAARrkHzreQKbFaKj0sYp950R0qPPcDeY6AE=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications/Itsycal.app
+    cp -R . $out/Applications/Itsycal.app
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Itsycal is a tiny menu bar calendar";
+    homepage = "https://www.mowglii.com/itsycal/";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ DontEatOreo ];
```

Comment:
```console
$ nix run nixpkgs#nixpkgs-review -- pr --print-result 299294
...
error: undefined variable 'DontEatOreo'

       at /Users/ysun/.cache/nixpkgs-review/pr-299294/nixpkgs/pkgs/os-specific/darwin/itsycal/default.nix:29:43:

           28|     license = lib.licenses.mit;
           29|     maintainers = with lib.maintainers; [ DontEatOreo ];
```

Please add an entry to https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix and backport it to your earliest contribution in nixpkgs where `lib.maintainers.DontEatOreo` is referenced.

I think it'd better if we involve a nixpkgs committer to see how we can resolve this, since #299032 was merged without `lib.maintainers.DontEatOreo` being defined.

Relevant: https://github.com/NixOS/nixpkgs/tree/master/maintainers

Reply:
I'm already there https://github.com/NixOS/nixpkgs/blob/aa1257d0d3751e2c65cc512382bdc1af1fd85c5c/maintainers/maintainer-list.nix#L5119-L5126

I guess it was a bit stupid of me, but didn't realize it was case-sensitive

Reply:
Ah, nice! I guess we would need a new PR to have the package introduced in #299032 to reference the correct maintainer (correct casing), and backport it

Reply:
Will do tomorrow

Reply:
Thanks for letting me know !